seqlike/SeqLike.py

```diff
@@ -187,14 +187,12 @@ def aa(self, auto_translate=True, **kwargs) -> "SeqLike":
         :param auto_translate: Whether to automagically translate
             an NT sequence into an AA sequence.
             Defaults to True.
-        :param kwargs: These kwargs are passed into
-            BioPython SeqRecord's `.translate()` method.
-            They can be used to set SeqRecord properties.
         :returns: copy of self with sequence object as amino acid sequence.
         """
         # Start with auto-translation
         if self._nt_record and self._aa_record is None and auto_translate:
-            return self.translate(**kwargs)
+            return self.translate(id=True, name=True, description=True,
+                                  annotations=True, dbxrefs=True)
```

Comment:
Then the implementation here can be changed:

```python
translate_kwargs = dict(
    id=True, name=True, 
    description=True,
    annotations=True,
    dbxrefs=True
)
translate_kwargs.update(kwargs)
return self.translate(**kwargs)
```

Reply:
Updated per your excellent suggestions @ericmjl-- thanks!